picoscope/ps4000a.py

```diff
@@ -278,28 +283,68 @@ def _lowLevelGetTimebase(self, tb, noSamples, oversample, segmentIndex):
 
     def getTimeBaseNum(self, sampleTimeS):
         """Return sample time in seconds to timebase as int for API calls."""
```

Comment:
This docstring makes no sense..... Can you think of a better one?

Reply:
ditto to below